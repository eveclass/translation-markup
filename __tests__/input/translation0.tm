LANGUAGES
  1: en-US
  2: pt-BR

SYSTEM
  ABOUT
    1: About
    2: Sobre
  COPY
    1: Copy
    2: Copiar
  DASHBOARD
    1,2: Dashboard
