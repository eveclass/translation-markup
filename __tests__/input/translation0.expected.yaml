LANGUAGES:
  1: english
  2: deutsch

HELLO:
  WORLD:
    1: Hello World
    2: Hallo Welt
